jlm/rvsdg/region.hpp

```diff
@@ -82,11 +80,17 @@ class RegionArgument : public output
    * @return A reference to the copied argument.
    */
   virtual RegionArgument &
-  Copy(rvsdg::Region & region, structural_input * input) = 0;
+  Copy(rvsdg::Region & region, structural_input * input);
 
   [[nodiscard]] std::variant<node *, Region *>
   GetOwner() const noexcept override;
 
+  static RegionArgument *
+  Create(
```

Comment:
Could you maybe add some documentation on top of it as well, since this is going to be THE method to add region arguments? Something along the lines of creating an argument and appending it to the region.

Reply:
added documentation